SwanContents/swancontents/filemanager/eos/fileio.py

```diff
@@ -107,58 +108,6 @@ def _get_os_path(self, path):
         else:
             return super()._get_os_path(path)
 
-    def _save_notebook(self, os_path, nb, capture_validation_error=None):
```

Comment:
So I understand it is fine to remove all this @diocas ?

Reply:
Confirmed by @PMax5 as a result of a conversation with @diocas  -- this was needed in the Puppet times.